src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -37,7 +39,7 @@ const getControls = memoize(
         meta: {
           label: 'Hoe komt de melding binnen?',
           path: 'source',
-          values: sources ? reduceSources(sources) : [],
+          values: sources ? reduceSources(filterSources(sources)) : [],
```

Comment:
Would be good to add a test for this change.

Reply:
with the next commit the can_be_selected prop is tested in wizard_step_1